frontend/packages/console-plugin-sdk/src/typings/storage-class-params.ts

```diff
@@ -0,0 +1,16 @@
+import { Extension } from './base';
+
+namespace ExtensionProperties {
+  export interface StorageClassProviosner {
+    getStorageClassProvisoner: () => any;
+  }
+}
+
+export interface StorageClassProviosner
```

Comment:
Nits ! fix everywhere
```suggestion
export interface StorageClassProvisioner
```